README.md

```diff
@@ -23,46 +23,46 @@ Fast open-source firmware for the [PineTime smartwatch](https://www.pine64.org/p
 
 ## Development
 
-- [InfiniTime Vision](doc/InfiniTimeVision.md)
-- [Rough structure of the code](doc/code/Intro.md)
-- [How to implement an application](doc/code/Apps.md)
+- [InfiniTime Vision](doc/dev/InfiniTimeVision.md)
+- [Rough structure of the code](doc/dev/code/Intro.md)
+- [How to implement an application](doc/dev/code/Apps.md)
 - [Generate the fonts and symbols](src/displayapp/fonts/README.md)
-- [Tips on designing an app UI](doc/ui_guidelines.md)
+- [Tips on designing an app UI](doc/dev/ui_guidelines.md)
 - [Bootloader, OTA and DFU](bootloader/README.md)
-- [External resources](doc/ExternalResources.md)
+- [External resources](doc/ref/ExternalResources.md)
```

Comment:
I wonder if this is actually the best way to organize the files. Many sections in the README includes files from both dev and ref. There are also some questions regarding the future direction of the documentation and the project in general, so it might be a bit early to be optimizing the documentation.

Reply:
Personally, I withheld from editing anything outside of the /doc/ folder except to fix links as I thought it'd be presumptuous to edit the readme :sweat_smile: 

What sort of questions are there? I'm more than willing to contribute to a larger-scale revamp including the readme, let me know if there's anything I could help with on that front.